src/backend/gog/games.ts

```diff
@@ -159,15 +159,8 @@ class GOGGame extends Game {
     const etaMatch = data.match(/ETA: (\d\d:\d\d:\d\d)/m)
     const bytesMatch = data.match(/Downloaded: (\S+) MiB/m)
     const progressMatch = data.match(/Progress: (\d+\.\d+) /m)
-    if (
-      etaMatch &&
-      etaMatch[1] &&
-      bytesMatch &&
-      bytesMatch[1] &&
-      progressMatch &&
-      progressMatch[1]
-    ) {
-      const eta = etaMatch[1]
+    if (bytesMatch && bytesMatch[1] && progressMatch && progressMatch[1]) {
+      const eta = etaMatch ? etaMatch[1] : null
```

Comment:
If we using only the index one, is it not enough to check for those?

Reply:
Just checking `bytesMatch[1]` for example?  
No, that would not be enough. `bytesMatch` itself can be null, and trying to access properties of null will throw a TypeError

Reply:
We can do `if (bytesMatch?.[1] && progressMatch?.[1])` which looks kinda ugly but it actually works xD